src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,19 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e:
+                    msg = ''
+                    for authzr in e.failed_authzrs:
+                       msg += f'\n Authorization for identifier {%s} failed.' % (
+                         authzr.body.identifier)
+                       msg += f'\n Here are the challenges that were not fulfilled:'
```

Comment:
This string does not have to a f-string as we are not adding something to it

Reply:
So what do you want?

This?:
```suggestion
                       msg += '\n Here are the challenges that were not fulfilled:'
```

Reply:
Yup